tests/test_metadata.py

```diff
@@ -0,0 +1,47 @@
+import datetime
+import json
+import math
+import os
+import time
+import pytest
+
+
+from coronavirus import MetadataRegion
+
+
+def test_MetadataRegion_basics():
+    m = MetadataRegion("Germany", "w")
+    # assert os.path.exists(MetadataStorageLocation)
+
+    m['html'] = "html-pfad"
+    m['ipynb'] = "ipynb-pfad"
+
+    m = MetadataRegion("UK", "w")
+    m['html'] = "html-path"
+
+    m = MetadataRegion("Germany")
+    assert m['html'] == "html-pfad"
+    assert m['ipynb'] == "ipynb-pfad"
+
+    m = MetadataRegion("UK")
+    assert m['html'] == "html-path"
+    with pytest.raises(KeyError):
+        m['missing-key'] 
```

Comment:
I don't fully understant the raise of KeyError here

Reply:
the `purest.raises()` notations is provided by `pytest` to check that (in this example) the `KeyError` is raised if the following block of commands are executed. 

(Another example is on slide 32 at https://www.desy.de/~fangohr/teaching/csds2019/pdfs/Software-Engineering-for-Computational-Science-and-Engineering-Hans-Fangohr.pdf, official documentation at https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions).

This test code is thus trying to access an entry which doesn't exist, and the test is meant to assert that in that case a `KeyError` is raised.